examples/tutorial_dbt_dagster_v2/tutorial_dbt_dagster_v2/assets/use_artifacts.py

```diff
@@ -0,0 +1,17 @@
+import pprint
+import sys
+
+from dagster import OpExecutionContext
+from dagster_dbt.asset_decorator import dbt_assets
+from dagster_dbt.cli import DbtCli
+
+from tutorial_dbt_dagster_v2.assets import manifest
+
+
+@dbt_assets(manifest=manifest)
+def my_dbt_assets(context: OpExecutionContext, dbt: DbtCli):
+    dbt_build = dbt.cli(["build"], manifest=manifest, context=context)
```

Comment:
in other places, we just have `yield from dbt.cli(...).stream()`

Reply:
This is an example to show how to retrieve artifacts from the dbt task after it is completed. We need a reference to the dbt task to retrieve the artifacts, which is why this deviates from the regular `yield from dbt.cli(...).stream()` syntax.